docs/custom-webview-android.md

```diff
@@ -9,28 +9,30 @@ Before you do this, you should be familiar with the concepts in [native UI compo
 
 ## Native Code
 
-To get started, you'll need to create a subclass of `ReactWebViewManager`, `ReactWebView`, and `ReactWebViewClient`. In your view manager, you'll then need to override:
+This example assumes you already have [react-native-community/react-native-webview](https://github.com/react-native-webview/react-native-webview) installed, if not please follow their [getting started guide](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Getting-Started.md) first.
```

Comment:
```suggestion
:::info
This example assumes you already have [`react-native-webview`](https://github.com/react-native-webview/react-native-webview) installed, if not please follow their [getting started guide](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Getting-Started.md) first.
:::
```

Let's use the package name there and wrap the sentence with note admonition. 🙂

Reply:
Done